src/sentry/relocation/tasks/transfer.py

```diff
@@ -82,7 +90,86 @@ def _find_relocation_transfer(
     silo_mode=SiloMode.CONTROL,
```

Comment:
Is it possible for a task like this to stall out longer than the retry wait duration? If so, does it make sense to give this task some upper bound as a timeout?

Reply:
It could happen if there were many concurrent relocations, as we could get stuck in a queue for longer than the retry reservation. That could be a problem in the future.